src/pages/students/[student_id].tsx

```diff
@@ -61,20 +56,32 @@ const ViewStudentPage = () => {
     onSuccess: () => utils.student.getStudentById.invalidate(),
   });
 
+  const editIepMutation = trpc.student.editIep.useMutation({
+    onSuccess: () => utils.student.getActiveStudentIep.invalidate(),
+  });
+
   const handleEditStudent = (e: React.ChangeEvent<HTMLFormElement>) => {
     e.preventDefault();
     const data = new FormData(e.currentTarget);
 
     if (!student) {
       return; // TODO: improve error handling
     }
+
     editMutation.mutate({
       student_id: student.student_id,
       first_name: data.get("firstName") as string,
       last_name: data.get("lastName") as string,
       email: data.get("email") as string,
       grade: Number(data.get("grade")) || 0,
     });
+
+    editIepMutation.mutate({
```

Comment:
I noticed that there is no checker to ensure that the end date is always after the start date. Do you want to add that logic in this PR or make it a separate issue?

Reply:
On the input for `endDate` line 303, if you put a `min={startDate}`, then you cannot select an end date before the start date. Also, lines 290 and 306 have reference to the states, but the states did not change, therefore those values remain empty strings. If you add this to your handleEditState, your values will populate the dates for the date picker.

`  const handleEditState = () => {
    setViewState(VIEW_STATES.EDIT);
    if (activeIep) {
      setStartDate(activeIep.start_date.toISOString().slice(0, 10));
      setEndDate((activeIep.end_date).toISOString().slice(0, 10));
    }
  };`

Reply:
![Screenshot 2024-04-04 at 9 46 23 PM](https://github.com/sfbrigade/compass/assets/104481165/94b9057a-6e32-4b5d-892f-5f0ee926f1e3)

Reply:
Thanks so much for that, @hieungo89 ! I added those initial values in the handleEdit and the min in the endDate, as well as an onChange in the startDate that keeps the endDate min dynamic.